### ISSUE TITLE
Upgrade to Helm v3.1.3

### ISSUE THREAD

**Issue by devth** [devth/helm-docker#89]
v3.1.3 released 2020-04-22T23:46:48Z
https://github.com/helm/helm/releases/tag/v3.1.3

**Comment by erSitzt** [comment]
does your build still work ?

i get the following error since `init` was removed in helm3
```
Step 13/20 : RUN helm init
 ---> Running in 411404150c3b
Error: unknown command "init" for "helm"
```

**Comment by devth** [comment]
@erSitzt I actually haven't been using it. Makes sense - I'll remove `init`.

**Comment by devth** [comment]
Removed it in 3.1.3, building now. Let me know if that doesn't work @erSitzt.

**Comment by Dbzman** [comment]
@devth Could you do the same for 3.2.0, please? :) It's also broken.
Thank you very much.

**Comment by devth** [comment]
@Dbzman just released a new 3.2.0 with the fix: https://hub.docker.com/layers/devth/helm/v3.2.0/images/sha256-12b4e7d00043f000542025c8292d4be39c55cccc9da84d0372d46678c99f47ff?context=explore
